setup.py

```diff
@@ -171,7 +171,8 @@ def write_version_py(filename='ompy/version_setup.py'):
  "uncertainties>=3.0.3",
  "tqdm",
  "pathos",
- "pybind11>=2.6.0"
+ "pybind11>=2.6.0",
+ "pymc3>=3.11.2,<4.0"
```

Comment:
What do you think about making pymc3, pymulintest (...) optional later on? -> So one could have a leightweight version of ompy reading and plotting spectra.

Reply:
I've thought about the same. I'm not sure exactly how this is best solved. Maybe that could be the goal for next major version of the OMpy project?